### PR TITLE
lesson-1-9

### PR DESCRIPTION
**Pull request by SvetlanaBeynik**
lesson-1-9 is completed.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -62,24 +64,32 @@ function App() {
...
-        ) : (
-          <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-        )}
-      </header>
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route
+          exact
+          path="/new"
```
Comment (viniciuspietscher) [inline src/App.js]: This route path should be only `"/"`

File: src/App.js [hunk 2]
```diff
@@ -62,24 +64,32 @@ function App() {
...
+          element={
+
+            <header>
+              <div style={{
+                // textAlign: 'left',
+                textAlign: 'center'
+              }}>
+                <h1>New Todo List</h1>
```
Comment (viniciuspietscher) [inline src/App.js:76]: Create a new route for `"/new"` and add this h1 to it

**Review by viniciuspietscher — CHANGES_REQUESTED** [review]
Hi Svetlana! Great job with the routing!

I requested some changes because we should have two routes, one for `"/"` that will have all your components, and another route for `"/new"` that will only have the h1 with "New Todo List".

At the end you should be able to visit `http://localhost:3000/` and see your list with the input field, and if you visit `http://localhost:3000/new` you will see only the h1 with "New Todo List".

Let me know on slack if you have any questions or need help.

File: src/App.js (hunk 1 above)
Comment (SvetlanaBeynik) [reply src/App.js]: The pull request is update

**Review by viniciuspietscher — APPROVED** [review]
Great job!!! Thank you for making the changes! 💯 🥇

**Comment by SvetlanaBeynik** [comment]
Lesson-1-9 done

**Pull request merged**